Golem/Yagna/YagnaApi.cs

```diff
@@ -111,9 +113,10 @@ public void Authorize(string key)
                 {
                     throw;
                 }
-                catch (Exception error)
+                catch (Exception e)
                 {
-                    _logger.LogError("Failed to get next stream event: {0}", error);
+                    _events.Raise(new ApplicationEventArgs("YagnaApi", $"Failed to get next stream event: {e.Message}", ApplicationEventArgs.SeverityLevel.Warning, e));
+                    _logger.LogError("Failed to get next stream event: {0}", e);
```

Comment:
We could have function on EventsPublisher, that does both in one call. It will probably simplify many places in the future